README.md

```diff
@@ -0,0 +1,262 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
```

Comment:
- [ ]  kindly consider customizing this title so that it is related to this current project to avoid confusion to the people visiting your readme